pyproject.toml

```diff
@@ -0,0 +1,83 @@
+[project]
+name = "bypass-url-parser"
+dynamic = ["version"]
+description = "Default template for PDM package"
+authors = [
+    {name = "Laluka", email = "loukajc@gmail.com"},
```

Comment:
Can you please add @jtof-fap & you here ? This improvement is a massive one to close the gap between "us hackers who give PoCs" and the real dev-eng world. That's warmly welcomed!

Reply:
Done! Note: Pypi currently only displays the first author.